crypto/asn1/i2d_pr.c

```diff
@@ -30,35 +31,23 @@ int i2d_PrivateKey(const EVP_PKEY *a, unsigned char **pp)
         }
         return ret;
     }
-    if (a->keymgmt != NULL) {
+    if (evp_pkey_is_provided(a)) {
+        /*
+         * This is horribly much, but then again, |*pp| is unbounded, and
```

Comment:
This comment makes no sense..

Reply:
How would you rephrase it?

Reply:
Its the horribly much bit..
something like..
|*pp| is unbounded, so just set an upper limit.

Reply:
Okie